### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by mmiszy**
- [x] update CLI invocation
- [x] update information about Storefront Example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,8 +73,8 @@ Learn more about [architecture](https://docs.saleor.io/docs/3.x/overview/archite
 - **GraphQL API**: Get many resources in a single request and [more](https://graphql.org/)
 - **Multichannel**: Per channel control of pricing, currencies, stock, product, and more
 - **Enterprise ready**: Secure, scalable, and stable. Battle-tested by big brands
-- **CMS**: Content is king, that's why we have a kingdom built-in
-- **Dashboard**: User friendly, fast, and productive. (Decoupled project [repo](https://github.com/saleor/saleor-dashboard) )
+- **CMS**: Content is king, that's why we have a kingdom built in
+**Dashboard**: User-friendly, fast, and productive. (Decoupled project [repo](https://github.com/saleor/saleor-dashboard) )
```
Comment (patrys) [inline README.md:77]: Leading bullet dash missing?

File: README.md [hunk 2]
```diff
@@ -73,8 +73,8 @@ Learn more about [architecture](https://docs.saleor.io/docs/3.x/overview/archite
 - **GraphQL API**: Get many resources in a single request and [more](https://graphql.org/)
 - **Multichannel**: Per channel control of pricing, currencies, stock, product, and more
 - **Enterprise ready**: Secure, scalable, and stable. Battle-tested by big brands
-- **CMS**: Content is king, that's why we have a kingdom built-in
-- **Dashboard**: User friendly, fast, and productive. (Decoupled project [repo](https://github.com/saleor/saleor-dashboard) )
+- **CMS**: Content is king, that's why we have a kingdom built in
```
[the remark this answers is not in the document]
Comment (mmiszy) [reply README.md:76]: Yes, these suggestions came from Grammarly :D

File: README.md (hunk 1 above)
Comment (mmiszy) [reply README.md:77]: Yes, I accepted Grammarly suggestions without double checking 🤦

File: README.md [hunk 3]
```diff
@@ -129,27 +129,18 @@ The easiest way to run all components of Saleor (API, storefront, and dashboard)
 
 ## Storefront
 
-An open-source storefront in React.js with Next.js. Built for Headless Commerce, using a modern stack with TypeScript, GraphQL, Apollo, and Tailwind CSS.
+An Open Source storefront example built with Next.js App Router, React.js, TypeScript, GraphQL, and Tailwind CSS.
```
Comment (maarcingebala) [inline README.md:132]: Open Source or open-source? We use two different ways of writing that in this file.

Comment (patrys) [reply README.md:132]: Open Source is written separately and capitalized when it refers to the Open Source Initiative as a noun. It's hyphenated in lowercase when used as an adjective (which I believe is the case here).

Comment (mmiszy) [reply README.md:132]: Okay, thank you! I'll update it here (and elsewhere if I find any occurrences)

**Review by aniav — APPROVED** [review]

**Review by patrys — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**